app/frontend/src/components/Navigation/Navigation.tsx

```diff
@@ -212,6 +212,8 @@ export default function Navigation() {
         <SearchBox />
         <div className={classes.bug}>
           <Hidden smDown>
+            <NavButton route={"https://couchers.org"} label={ABOUT} />
+            <NavButton route={"https://community.couchers.org"} label={FORUM} />
```

Comment:
Instead of passing the route like this, for best practice you should define the route in `routes.ts` and import it at the top with the other routes, then pass it as a variable.

Reply:
This makes sense, thank you for the advice! I made the change in the latest commit.